README.md

```diff
@@ -154,6 +154,28 @@ Composite index keys are not supported.
 
 The primary key isn't indexed by default.
 
+## Serialized fields
+
+A custom class can be used to deserialize an attribute.  By default `CustomClass.new(value)`, is called where
+`value` is the raw value from the source file.  A method other than `new` can be called by passing a
+`:deserializer_method` name as a symbol.
+
+
+```ruby
+class ContinentString < String ; end
+class CurrencyCodeEnum < T::Enum
```

Comment:
Please don't use `T::` stuff in code sample, that require knowledge of sorbet to understand.

Reply:
Fixed!